packages/flame/test/game/game_render_box_test.dart

```diff
@@ -9,6 +9,8 @@ class _MockFlameGame extends Mock implements FlameGame {}
 
 class _MockBuildContext extends Mock implements BuildContext {}
 
+// Ignoring dart linting error due
+// ignore: subtype_of_base_or_final_is_not_base_final_or_sealed,invalid_use_of_type_outside_library
```

Comment:
This was on the main channel right? I think we can remove it until it is released.